.github/workflows/cabal-ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
           submodules: true
       - uses: haskell/actions/setup@v2
         with:
-          ghc-version: '9.6.1'
-          cabal-version: '3.8.1.0'
+          ghc-version: '9.6.3'
+          cabal-version: '3.10.1.0'
```

Comment:
Would be best to use specify version 3.10.2.0 just in case there are some fixes we could benefit from.